README.md

```diff
@@ -8,39 +8,56 @@ will likely not be included in this repository, such as custom webhooks, etc.
 
 See also https://wikitech.wikimedia.org/wiki/Wikimedia_Cloud_Services_team/EnhancementProposals/Toolforge_Buildpack_Implementation
 
+## Setup dev environment
 
-### Setup dev environment
-#### Requirements
+### Requirements
 You will need to install:
 * [docker](https://www.docker.com/) -> as the container engine for everything.
 * [docker-compose](https://docs.docker.com/compose/) -> for harbor (docker registry).
 * [minikube](https://minikube.sigs.k8s.io/docs/) -> as dev kubernetes installation.
 
-#### Run harbor
-You can install it with the helper script:
-* `utils/get_harbor.sh`
-After that you can use docker-compose to run the whole harbor system (see the output of the script for the exact command).
-
-Once harbor is running, you will need to make sure that it's setup (user project created and such) so run:
-- `utils/setup_harbor.py`
-
 ### Setup minikube
 We will need to get a specific k8s version (1.21.8 is the current toolforge version when writing this, you might want to double check):
  - `minikube start --kubernetes-version=v1.21.8`
 
+### Find your Docker IP
+You will need to reach the Docker daemon on your host from inside Minikube using
+an IP (this is a limitation of Tekton that doesn't support hostnames unless you
+have a valid SSL certificate).
```

Comment:
It's not the ip of the docker daemon, it's the ip where harbor is exposed no?

Reply:
Yes, though it's a bit counterintuitive as you have to know it before Harbor is even installed 😛 In your setup, it's not the Docker IP but the host IP, I'm trying to think what could be a generic way to word this...

Reply:
Hmm, I think something like this might cover both:

### Find you future local Harbor IP 
You will need to reach your local Harbor instance from inside Minikube using an IP (this is a limitation of Tekton that doesn't support hostnames for self-signed certificates).

The IP to use will vary depending on your machine and OS, as we deploy harbor with docker-compose this will be the IP where docker lives.

This is usually:
- `minikube ssh`
- `grep host.minikube.internal /etc/hosts`

But sometimes (ex. debian bullseye) you will need to use the external IP of your host:
- `hostname -I| awk '{print $1}'`

You will be able to verify it later when setting up Harbor.

> then once harbor is up in the next section

You can verify if the IP you chose works by curling to it:

```
>$ curl -s http://$(hostname -I| awk '{print $1}') -o /dev/null && echo "works" || echo "does not work"
works
```

If it does not, use the other alternative method or select one of the different IPs from your laptop (and/or ask for help!)